inc/toolbox.class.php

```diff
@@ -307,6 +307,22 @@ public static function getGlpiItemtypes(): array
             __('Other')                                               => $other_itemtypes,
         ];
 
+        $plugin = new Plugin();
+        if ($plugin->isActivated('genericobject') && method_exists('PluginGenericobjectType', 'getTypes')) {
+            $go_itemtypes = [];
+            foreach (array_keys(PluginGenericobjectType::getTypes()) as $go_itemtype) {
+                if (!class_exists($go_itemtype)) {
+                    continue;
+                }
+                $go_itemtypes[] = $go_itemtype;
+            }
+            if (count($go_itemtypes) > 0) {
+                $all_itemtypes += [
+                    $plugin->getInfo('genericobject', 'name') => $go_itemtypes,
+                ];
```

Comment:
It seems a bit unnecessary to use an "array union" to add a single element.

```suggestion
                $key = $plugin->getInfo('genericobject', 'name');
                $all_itemtypes[$key] = $go_itemtypes;
```